CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Support `IPROTO_FEATURE_SPACE_AND_INDEX_NAMES` for Tarantool
   version >= 3.0.0-alpha1 (#338). It allows to use space and index names 
   in requests instead of their IDs.
+- `GetSchema` method to the `Connection` struct (#7)
```

Comment:
No, let's just make this a separate function.

```Go
func GetSchema(conn tarantool.Connector) (tarantool.Schema, error)
```

Reply:
Made `GetSchema` a separate function. Fixed docs in `README` and `CHANGELOG`. Added an example on how to update the connection schema.